src/app/categories/categories.component.html

```diff
@@ -8,7 +8,7 @@ <h1 class="text-center text-info font-weight-bold">Categories</h1>
         <div class="card">
           <div class="fa-7x fas fa-dog" id="category-icons">
             <div class="w-100">
-              <div class="btn btn-info m-2">
+              <div class="btn btn-info m-2" id="category-button">
```

Comment:
I noticed that all of the ids are category-button, best practice is that all ids should be unique